From: SeongJae Park <sj@kernel.org>
Date: Tue, 25 Feb 2025 13:40:29 -0800
Subject: [PATCH] ==== selftest fixes ====

selftest/damon: fixes and an improvement

Fix three DAMON selftest bugs that causes false positive failures and
success, and make an improvement of DAMON test-purpose sysfs wrapper for
better debugging and future tests.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus_dxc_h0u_ | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus_dxc_h0u_

diff --git a/hkml_cv_bogus/hkml_cv_bogus_dxc_h0u_ b/hkml_cv_bogus/hkml_cv_bogus_dxc_h0u_
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.5

